src/modality-specific-files/magnetic-resonance-imaging-data.md

```diff
@@ -1357,3 +1362,265 @@ As for other EPI sequences, these field mapping sequences may have any of the
 [in-plane spatial encoding](#in-and-out-of-plane-spatial-encoding) metadata keys.
 However, please note that `PhaseEncodingDirection` and `TotalReadoutTime` keys
 are REQUIRED for these field mapping sequences.
+
+### Radiofrequency (RF) field mapping
+
+Fieldmaps may be acquired to measure the inhomogeneity in the RF (B<sub>1</sub>) field.
+These fieldmaps may be divided into two categories: transmit (B<sub>1</sub><sup>+</sup>) and receive (B<sub>1</sub><sup>-</sup>) fieldmaps.
```

Comment:
Out of curiosity, does mkdocs handle $\LaTeX$?

```suggestion
Fieldmaps may be acquired to measure the inhomogeneity in the RF ($B_1$) field.
These fieldmaps may be divided into two categories: transmit ($B_1^+$) and receive ($B_1^-$) fieldmaps.
```

Reply:
<img width="1316" height="252" alt="image" src="https://github.com/user-attachments/assets/5a9493b4-6c2e-455c-a19d-8c00492ad567" />

I think there might be extensions we could use to enable LaTeX support, but the current configuration doesn't seem to support it.